code/game/objects/items/weapons/clown_items.dm

```diff
@@ -36,9 +36,10 @@
 	cleanspeed = 10 //much faster than mop so it is useful for traitors who want to clean crime scenes
 
 /obj/item/weapon/soap/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] crams the bar of soap down \his throat! It looks like \he's trying to commit suicide..</span>")
-	. = TOXLOSS
-	src.loc = user
+	user.say(";FFFFFFFFFFFFFFFFUUUUUUUDGE!!")
+	user.visible_message("<span class='suicide'>[user] lifts the [src.name] to their mouth and gnaws on it furiously, producing a thick froth! They'll never get that BB gun now!")
+	PoolOrNew(/obj/effect/effect/foam, loc)
+	return (TOXLOSS)

```

Comment:
There's a popular film which plays on numerous television stations in the United States every year called 'A Christmas Story.' 

In it, the main character, Ralphie, pines after a Red Rider BB Gun, wishing for it more than anything in the whole wide world.

At one point, due to an event occurs which leads Ralphie to say "OH FUUUUUUDDDDDGE" except he doesn't say Fudge in real life, just in the movie. He makes sure to tell you this.

As a result, he gets his mouth washed out with soap. Because he has a horrible, horrible dirty mouth.

It's a pretty iconic movie and scene. You should watch it, and then spend the rest of your life avoiding having to watch it EVERY YEAR, MULTIPLE TIMES.